### DISCUSSION TITLE
Firebird driver missing?

### DISCUSSION THREAD

**Discussion by wmeyer48** [dbeaver/dbeaver#21717] [Q&A] [answered]
### Description

In a new VM, I have installed Firebird 4, and DBeaver 23.2.3. When I try to connect to my Firebird DB, DBeaver reports that driver files are missing. However, that dialog, which has a checkbox to force driver download, is immediately overlaid by another modal dialog, so the download option is useless.
In the Driver Manager dialog, Firebird shows as present.
I have limited experience with DBeaver, so please be as clear as possible in responses.

### DBeaver Version

Community Edition 23.2.3.202310221533

### Operating System

Windows 10 64-bit

### Database and driver

Firebird 4.0.3
Driver installed by FB, I would guess, or by DBeaver. Can't check in connection dialog, as it throws this error:
![DBeaverDriver](https://github.com/dbeaver/dbeaver/assets/19627715/463d8a98-8900-43a9-ad96-3cbf98bbe26c)

### Steps to reproduce

1. I try to connect to DB.
2. It reports Firebird driver files missing.
I have not discovered whether there is some way from within DBeaver to download the drivers

**Comment by E1izabeth** [comment, marked as answer]
Please try switching the setting mentioned in the message

**Comment by wmeyer48** [comment]
I did that. It made no difference yesterday. But I installed 22.2.1 yesterday, and this morning let it upgrade to 23.2.3, then changed the setting again, and it offered to install the driver. Thank you.

**Comment by E1izabeth** [comment]
Thank you for update
